content/posts/manage-list-feature-flags-with-file/index.md

```diff
@@ -0,0 +1,229 @@
+    strategyName: strategy_organization
+    parameters: {}
+    constraints: []
+  - featureName: feature_yyyyy_v1
+    projectId: default
+    environment: development
+    strategyName: strategy_organization
+    parameters: {}
+    constraints: []
+  - featureName: feature_yyyyy_v1
+    projectId: default
+    environment: production
+    strategyName: strategy_organization
+    parameters: {}
+    constraints: []
+environments:
+  - name: default
+    type: production
+    sortOrder: 1
+    enabled: false
+    protected: true
+  - name: development
+    type: development
+    sortOrder: 100
+    enabled: true
+    protected: false
+  - name: production
+    type: production
+    sortOrder: 200
+    enabled: true
+    protected: false
+  - name: staging
+    type: production
+    sortOrder: 300
+    enabled: true
+    protected: false
+featureEnvironments:
+  - enabled: true
+    featureName: feature_yyyyy_v1
+    environment: development
+  - enabled: true
+    featureName: feature_yyyyy_v1
+    environment: production
+  - enabled: true
+    featureName: feature_xxxx_v1
+    environment: development
+  - enabled: true
+    featureName: feature_xxxx_v1
+    environment: production
+```
+In this file, we have: 
+
+`features`: list of [feature toggles](https://docs.getunleash.io/user_guide/create_feature_toggle), a feature toggle includes *description*, *type*,[*variants*](https://docs.getunleash.io/advanced/toggle_variants).... If a toggle has *variants*, then the *variants* object can also contain an optional payload property. The payload will contain data about the variant's payload: what type it is, and what the content is.  In this example, there are 2 feature toggles *feature_xxxx_v1* and *feature_yyyyy_v1*.
+
+Note: *Unleash uses a fallback variant called "disabled" to indicate that a toggle has no variants. However, you are free to create a variant called "disabled" yourself. In that case you can tell them apart by checking the variant's enabled property: if the toggle has no variants, enabled will be false. If the toggle is the "disabled" variant that you created, it will have enabled set to true.*
+
+`strategies`: you can define your list of custom [*strategies*](https://docs.getunleash.io/user_guide/activation_strategy). Unleash comes with a few common activation strategies. Some of them require the client to provide the unleash-context, which gives the necessary context for Unleash.In this example, I create a strategy named *strategy_organization*
+
+`environments`: list of custom [*environments*](https://docs.getunleash.io/user_guide/environments).Environments is a new way to organize activation strategy configurations for feature toggles into separate environments. In Unleash, a feature lives across all your environments — after all, the goal is to get the new feature released as soon as possible — but it makes sense to configure the activation differently per environment.(Environments are available in Unleash v4.3.x and later)
```

Comment:
`environments`: list of custom [*environments*](https://docs.getunleash.io/user_guide/environments).Environments is a new way to organize activation strategy configurations for feature toggles into separate environments. In Unleash, a feature lives across all your environments — after all, the goal is to get the new feature released as soon as possible — but it make sense to configure the activation differently per environment.(Environments are available in Unleash v4.3.x and later)